modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCPrimaryKeyFinderOnlineTest.java

```diff
@@ -39,6 +40,7 @@ protected void connect() throws Exception {
         }
     }
 
+    @Test
```

Comment:
I don't think this is correct, the test in question extends from Junit3 TestCase (all OnlineTest do, there is a ton in the coebase):

![image](https://user-images.githubusercontent.com/325433/54864835-f2f54100-4d5c-11e9-9300-cba45b9cbec8.png)

Migrating them to Junit 4 would be nice, at the same time, a ton of work :-D
If you feel like migrating some, it would certainly be appreciated.

Reply:
May also be causing the build failues, all builds on Travis and Appveyor are complaining.

Reply:
Sorry I missed that. I am certain I've seen that combination in use on some other tests.
Don't have much experience in migrating to JUnit 4, but I do have some knowledge on the good old JUnit3 :)
If I get some extra time to spend I could look into it. 

I get some test errors running the jdbc tests against Oracle. Are those tests part of Travis and if so, what version of Oracle DB?

Reply:
With the exclusion of H2 tests, Travis does not normally run database tests. If the H2 tests fail (I believe they do),  it's likely a generalized condition.